crates/lib/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "liquid-lib"
+version = "0.19.1"
+authors = ["Johann Hofmann <mail@johann-hofmann.com>"]
+description = "The liquid templating language for Rust"
+repository = "https://github.com/cobalt-org/liquid-rust/tree/master/liquid-lib"
+documentation = "https://docs.rs/liquid-lib"
+readme = "README.md"
+categories = ["template-engine"]
+keywords = ["liquid", "template", "templating", "language", "html"]
+license = "MIT"
+edition = "2018"
+
+[package.metadata.docs.rs]
+features = [ "default", "jekyll", "all" ]
+
+[badges]
+azure-devops = { project = "cobalt-org", pipeline = "liquid-rust" }
+
+[dependencies]
+liquid-core = { version = "^0.19", path = "../core" }
+kstring = { version = "0.1.0", path = "../kstring" }
+itertools = "0.8.0"
+regex = "1.0"
+unicode-segmentation = "1.2"
+url = "1.5"
+chrono = "0.4"
+once_cell = "1.0"
+deunicode = { version = "1.0.0", optional = true }
+
+[features]
+default = ["stdlib"]
+stdlib = []
+jekyll = ["deunicode"]
+all = []

```

Comment:
Should we add a `shopify` for pluralize and friends?

https://shopify.dev/docs/liquid/reference/filters/string-filters

Reply:
Are all the stuff in `extra` shopify related?
if yes, maybe just rename `all` to `shopify`?

Reply:
I don't know if everything in extra lines up with shopify.

As for all, we'd still need that since std, shopify, and jekyll each have different sets.

Reply:
I'm not sure to what `all` refers to then. 
- shopify for Shopify specific stuff
- jekyll for Jekyll specific stuff
- std for standard liquid (I though it was defined by Shopify)
and… all?

Reply:
All would just enable shopfiy, jekyll, std, and extra.

Reply:
ok :)
It seem the other stuff in extra is not shopify related, it's DateInTz stuff and there is this comment: // liquid-rust proprietary

So I'll create shopify dir, and move pluralize in